docs/sql/statements/revoke.rst

```diff
@@ -19,17 +19,24 @@ Synopsis
 
   REVOKE { { DQL | DML | DDL | AL [,...] } | ALL [ PRIVILEGES ] }
   [ON {SCHEMA | TABLE | VIEW} identifier [, ...]]
-  FROM user_name [, ...];
+  FROM name [, ...];
+
+  REVOKE role_name_to_revoke [, ...] FROM name [, ...]
 
 Description
 ===========
 
-``REVOKE`` is a management statement to revoke previously granted privileges
-on a specific object from one or many existing users.
+``REVOKE`` is a management statement which comes in two flavours.
 
+The first one is used to revoke previously granted privileges on a specific
+object from one or many existing users or roles.
 ``ON {SCHEMA | TABLE | VIEW}`` is optional, if not specified the privilege will
 be revoked on the ``CLUSTER`` level.
 
+The second one is used to revoke previously granted roles from one or many
+existing users or roles. Thus, the users or roles loose the privileges which
+had automatically inherit from those previously granted roles.
+
```

Comment:
1. let's link insert to :ref:granted`<roles_inheritance>`
2. can we add a note/example that revoking a privilege might be not enough if this privilege is inherited via multiple parent roles?

Say user has role "reader" with DQL and role "writer" with DQL+DML. 
If we revoke "reader"  from user, DQL won't be lost because it's still possessed via "writer"